src/components/BasicAuctionClientPage/index.tsx

```diff
@@ -12,8 +12,10 @@ import * as S from './BasicAuctionClientPage.css';
 function BasicAuctionClientPage() {
   const searchParams = useSearchParams();
   const searchKeywordParam = searchParams.get(AUCTIONPARAM_KEY.Q);
+  const pickCategory = Number(searchParams.get('categoryId'));
 
   const { data } = useGetBasicAuctionList({
+    categoryId: pickCategory || undefined,
```

Comment:
검색 할때 categoryId=undefined인 상태로 요청이 되는데 쿼리 스트링에서 undefined나 빈값들을 필터링해서 요청하는 로직이 필요해 보여요!

Reply:
```suggestion
    categoryId: pickCategory || '',
```
타입을 조금 수정하여 빈 문자열로 넘겨주었습니다!